### PR TITLE
Updates to README and Makefile's in the GPU package

### PR DESCRIPTION
**Pull request by ndtrung81**
**Summary**

Update README and the Makefile's in lib/gpu with -L$(CUDA_HOME)/lib64/stubs so that libcuda.so in the recent versions of the CUDA toolkit can be found during linking

**Author(s)**

Trung Nguyen (Northwestern)

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

N/A

**Implementation Notes**

N/A

**Post Submission Checklist**

_Please check the fields below as they are completed **after** the pull request has been submitted. Delete lines that don't apply_

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [ ] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [x] The feature has been verified to work with the conventional build system
- [ ] The feature has been verified to work with the CMake based build system
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

### CODE REVIEW

**Comment by rbberger** [comment]
The same issue happens with CMake. Not sure if we want to add something equivalent to the CMake build or just document the workaround by setting `CMAKE_LIBRARY_PATH` or `CUDA_CUDA_LIBRARY` directly. Comments?

**Comment by ndtrung81** [comment]
I would go with documenting the workaround by setting CUDA_CUDA_LIBRARY directly.

File: lib/gpu/Makefile.fermi [hunk 1]
```diff
@@ -4,7 +4,7 @@ EXTRAMAKE = Makefile.lammps.standard
 CUDA_ARCH = -arch=sm_35
 CUDA_PRECISION = -D_SINGLE_DOUBLE
 CUDA_INCLUDE = -I$(CUDA_HOME)/include
-CUDA_LIB = -L$(CUDA_HOME)/lib64 -Xlinker -rpath -Xlinker $(CUDA_HOME)/lib64 -lcudart
+CUDA_LIB = -L$(CUDA_HOME)/lib64 -Xlinker -rpath -Xlinker $(CUDA_HOME)/lib64/stubs -lcudart
```
Comment (akohlmey) [inline lib/gpu/Makefile.fermi]: This change is incorrect. You do **not** want to encode the CUDA stubs folder as rpath into an executable, you only want to use the stub folder for linking. At runtime you want the real libraries, not the stubs.

Comment (ndtrung81) [reply lib/gpu/Makefile.fermi]: You are right, sorry for the mistake.

**Pull request merged**